standalone/test/unit_tests/util/unittest_tenth_of_degree.cpp

```diff
@@ -135,9 +135,7 @@ TEST(TenthOfDegreeTest, LargerThanOrEqualComparison)
 
 TEST(TenthOfDegreeTest, shouldThrowUnderflowErrorOnCastToIntOfValuesUnderZero)
 {
-  EXPECT_THROW(std::cout << static_cast<uint16_t>(util::TenthOfDegree(-1))
-                         << " - printed to not be unused. Should throw before!\n",
-               std::underflow_error);
+  EXPECT_THROW(static_cast<uint16_t>(util::TenthOfDegree(-1)), std::underflow_error) << "Should throw underflow error";
```

Comment:
Clang complains. Maybe the static_cast could be droped as well? 
```suggestion
  EXPECT_THROW(util::TenthOfDegree(-1), std::underflow_error) << "Should throw underflow error";
```
And in case the warning persists (see https://nelkinda.com/blog/suppress-warnings-in-gcc-and-clang/#w1223aab3b1c57):
```suggestion
  EXPECT_THROW(__attribute__((unused)) util::TenthOfDegree(-1), std::underflow_error) << "Should throw underflow error";
```

Reply:
The solution needs to build with MSVC as well

Reply:
I think removing the cast removes a bit clarity about what is tested